data/Place.csv

```diff
@@ -53,24 +53,24 @@ PL0051,Changsha,28.228209,112.938814,2017-07-12,OS,2017-07-12,OS,
 PL0052,Miluo,28.806394,113.06724,2017-07-12,OS,2017-07-12,OS,
 PL0053,Beidahuang,45.73722,126.692441,2017-07-12,OS,2017-07-12,OS,
 PL0054,Urumqi,43.825592,87.616848,2017-07-12,OS,2017-07-12,OS,
-PL0055,Xichang,27.894504 ,102.264449 ,2017-07-12,OS,2017-07-12,OS,
+PL0055,Xichang,27.894504,102.264449,2017-07-12,OS,2017-07-12,OS,
 PL0056,Jinjiang (Fujian),24.781681,118.552365,2017-07-18,LH,2017-07-18,LH,
 PL0057,Nizhny Novgorod,56.296504,43.936059,2017-07-19,LH,2017-07-19,LH,
-PL0058,Saint Petersburg,59.934280,30.335099,2017-07-18,OS,2017-07-18,OS,
+PL0058,Saint Petersburg,59.93428,30.335099,2017-07-18,OS,2017-07-18,OS,
 PL0059,Mojkovac,42.9586111111,19.5816666667,2017-07-25,MG,2017-07-25,MG,
 PL0060,Changge,34.2082008339,113.757755302,2017-08-01,LH,2017-08-08,MG,
 PL0061,Lufeng,23.165614,116.210632,2017-08-08,MG,2017-08-08,MG,
-PL0062,Yanting,31.208363,105.389453 ,2017-08-11,MG,2017-08-11,MG,
-PL0063,Berlin,52.5243700,13.4105300,2017-08-15,MG,2017-08-15,MG,
+PL0062,Yanting,31.208363,105.389453,2017-08-11,MG,2017-08-11,MG,
+PL0063,Berlin,52.52437,13.41053,2017-08-15,MG,2017-08-15,MG,
 PL0064,Vietnam,14.058324,108.277199,2017-08-26,MG,2017-08-26,MG,
-PL0065,Prague,50.075538,14.437800,2017-07-19,OS,2017-07-19,OS,
+PL0065,Prague,50.075538,14.4378,2017-07-19,OS,2017-07-19,OS,
 PL0066,Lowell,42.633425,-71.316172,2017-07-20,OS,2017-07-20,OS,
 PL0067,Ganzhou,25.830694,114.933546,2017-08-31,MG,2017-08-31,MG,
 PL0068,Nuoro,40.320242,9.326438,2017-09-06,CW,2017-09-06,CW,
-PL0069,Ningbo,29.868336,121.543990,2017-09-06,CW,2017-09-06,CW,
-PL0070,Guangzhou,23.128994 ,113.253250,2017-09-08,MG,2017-09-08,MG,
-PL0071,,,,,,,,
-PL0072,,,,,,,,
+PL0069,Ningbo,29.868336,121.54399,2017-09-06,CW,2017-09-06,CW,
+PL0070,Guangzhou,23.128994,113.25325,2017-09-08,MG,2017-09-08,MG,
+PL0071,Lanzhou,36.061089,103.834303,2017-09-16,CW,2017-09-16,CW,
```

Comment:
There is a non-tivial conflict here, since these IDs have already been assigned to other places in the main branch. Please make sure that no entries in other tables reference these places when resolving the conflicts

Reply:
Ok! But I haven't tried to solve conflicts before, so I've got some questions: 
- Do I stay in my "16.09" branch and change the data files or do I create a new branch?
- How to see the Master data so that I can do solve the conflict according to the latest version? (I assume the data in my explore is the ones that I made changes to) 
Hope they are not silly questions, thanks!

Reply:
Yes you solve the conflicts in this branch (16.09). Any commits here will be automatically added to the PR. You can read more about solving conflicts via gitHub help. No matter where you view the conflicts you ll see both changes from master and your branch to make your selection. In this particular case you can simple use search (top left on the browser) on the data folder of the master branch to see any occurrences of the IDs in question. If there are no mentions then you can simple modify the IDs, so there are no more duplicates. If there are mentions of the ID you ll need to cross check and make sure what they refer to .